src/client/containers/ProfileHolder.jsx

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux'
+import Profile from '../components/Profile'
+
+const mapStateToProps = (state) => ({
+  user: state.user,
+  viewUser: state.viewUser,
+});
+
+// changeViewUser,
+const mapDispatchToProps = ({
+});
```

Comment:
Is mapDispatchToProps a placeholder to wrap future action creators? Otherwise, can be removed.